index.html

```diff
@@ -23,5 +24,152 @@ <h1 class="playfairFont">José Fernández-Caminero</h1>
   </main>
 
   <section class="middleDeco"></section>
+
+  <section>
+    <h2 class="playfairFont padding10twenty">Projects</h2>
+
+    <div class="dgrid">
+      <img class="fullWidth" src="resources/Rectangle 21.png" alt="Laptop, tangerine, and other objects">
+      <div class="darkBlueBackground projectsPadding">
+        <p class="projectnameStyle">Project name goes here</p>
+  
+        <ul class="greenul">
+          <li class="greenli">HTML/CSS</li>
+          <li class="greenli">Ruby on Rails</li>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Cool (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+
+      <img class="fullWidth" src="resources/macbook.png" alt="A macbook">
+      <div class="darkBlueBackground projectsPadding">
+        <p class="projectnameStyle">Project name goes here</p>
+  
+        <ul class="greenul">
+          <li class="greenli">HTML/CSS</li>
+          <li class="greenli">Ruby on Rails</li>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Cooler (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Cooler (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+
```

Comment:
- [ ] As required for the project, kindly make sure the content of each project card is aligned with the `flexbox`

Reply:
![projectinstructions](https://user-images.githubusercontent.com/49853028/227654483-ce2e5bc2-ba3e-48da-afc5-694075d8d2e8.png)

![cardflexbox](https://user-images.githubusercontent.com/49853028/227654516-888c6f6b-f6bd-4c79-88f6-8bba0a1542fb.png)

Hi @tresorsawasawa Are you sure about this one too? I used CSS Grid on the main container (just like it says in the instructions). And I used flexbox on the cards, on the only part of the cards that needed it. It seems to me that flexbox is useless anywhere else. Am I confused? Thanks!

Reply:
Hi @fdezcaminero 

This comment means that the all elements of each card should be styled using flexbox not only langages inside the card. The elements of a card are: image, Project name, used languages and the See Project button